virttest/virsh.py

```diff
@@ -713,9 +712,16 @@ def _get_event_output(session):
 
             if wait_for_event is True and event_type is not None:
                 virsh_session = EventTracker.start_get_event(str(args[0]))
-                # Sometimes the output of virsh_session is missing, add a sleep
-                # for debug. Will remove it if it is unnecessary.
-                time.sleep(3)
+                # Sometimes the output of session can't be gotten immediately.
+                # Wait for a while to avoid this situation.
+                if not utils_misc.wait_for(
+                        lambda: re.search(
+                            "Welcome to virsh",
+                            _get_event_output(virsh_session)
+                        ), 3):
```

Comment:
Maybe you can extend the timeout to a little longer, like 10s

Reply:
> Maybe you can extend the timeout to a little longer, like 10s

I think it should not take 10s to get output from virsh session. Otherwise there maybe a regression I think.

Reply:
@liuyd96 , I suggest timeout can be 10 seconds considering some extreme cases on variant platform arch